### ISSUE TITLE
start up and then, nothing

### ISSUE THREAD

**Issue by risk22** [CIRCL/Circlean#67]
Any known issues with Rpi3? It boot and goes for 5-10sek and then dies. Same image works on Rp1 Gen1.

**Comment by Rafiot** [comment]
The rpi3 is supported. 

Did you plug the 2 USB keys before booting the rpi?

**Comment by risk22** [comment]
Yes, both plugged in.. same img works on rpi gen1
On rpi3 it’s booting but dies after approx 10sec

Thanks


sön 9 sep. 2018 kl. 22:03 skrev Raphaël Vinot <notifications@github.com>:

**Comment by Rafiot** [comment]
So that's very similar to the bug I had with some keys that take a while to be recognized by the OS, but that was supposedly fixed by the 10s waiting time. I guess there is something else going on... Did you try with other USB keys?

**Comment by nicoladiaz** [comment]
Hello,
I tried few minutes ago with raspi 1 & 3 and same thing no copy at all from one USB key to the other one :/ 
Meanwhile i can hear the music (randomly TheCure, RATM :)
Img used > 2018-07-23-CIRCLean.img

**Comment by Rafiot** [comment]
if you were hearing music, it was working. Can you let it plugged for longer? Depending on the files you're working on it can take a while: rpis are USB2 so everything takes forever.

**Comment by owandus** [comment]
Hello there, 

I am having exact same issue with rpi3, tested with a different keys and waited longer after the music stops. Any idea of the issue's cause?

Actually when the music stops, the rpi switches immediately off. Is it the expected behavior?

Thank you in advance!

**Comment by Rafiot** [comment]
Yes, it is the expected behavior. The music starts when the processing of the files begins, and stops when the rPi shutdowns.

Was the destination key empty?

**Comment by Rafiot** [comment]
This is super weird... I had issues with the rpi 3 booting faster than the USB keys were recognized and that was causing a similar issue... but the timer I added should have fixed it (and it did with the set of keys I used). Sorry for that, I'll try to find a more robust solution.
